### PR TITLE
API

### PR DESCRIPTION
**Pull request by brytebee**
## Connect with the API 🌐🌐
In this PR, I have accomplished the following:
- attached my app to an API
- added book to the API
- removed book from the API

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @brytebee 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/book.js [hunk 1]
```diff
@@ -1,24 +1,69 @@
-const ADD_BOOK = 'bookStore/books/ADD_BOOK';
-const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+const ADD_BOOK = 'bookStore-react/books/ADD_BOOK';
```
Comment (mikenath223) [inline src/redux/books/book.js:1]: - [x] There is an issue in adding a book it fails with this error: `Invalid prop book.item_id`. Kindly fix it.

Comment (brytebee) [reply src/redux/books/book.js:1]: Done! Books can be added easily now 👍

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by brytebee** [comment]
Hello, @arslanbisharat 👋👋,

Thanks for the warm-hearted review 👀👀.



I'll go ahead to merge and move on to the next milestone 🏁🏁.

**_Happy Hacking 😄😄_**

**Pull request merged**